frontend/templates/product/sections/ProductOverviewSection/index.tsx

```diff
@@ -219,12 +219,25 @@ export function ProductOverviewSection({
                         product.compatibility.devices.length <= 0
                      }
                   >
-                     <CustomAccordionButton>
-                        Compatibility
-                     </CustomAccordionButton>
-                     <CustomAccordionPanel data-testid="product-compatibility-dropdown">
-                        <CompatibleDevices product={product} />
-                     </CustomAccordionPanel>
+                     {hasCompatibilityNotes ? (
+                        <>
+                           <CustomAccordionButton>
+                              Compatibility Notes
+                           </CustomAccordionButton>
+                           <CustomAccordionPanel>
+                              {product.compatibilityNotes}
+                           </CustomAccordionPanel>
+                        </>
+                     ) : (
+                        <>
+                           <CustomAccordionButton>
+                              Compatibility
+                           </CustomAccordionButton>
+                           <CustomAccordionPanel data-testid="product-compatibility-dropdown">
+                              <CompatibleDevices product={product} />
+                           </CustomAccordionPanel>
```

Comment:
We might consider mirroring the UI choices made for the `Compatibility` data. This section shows a truncated list of 3 devices, and optionally includes a line with "And X more devices...". It also is a hyperlink to the compatibility section below.

Reply:
Yeah I think something like this would work well here to handle long lists. We should split on newline, trim each row, filter empty rows, and then join on `, ` before printing. The first 3 (or 10 or whatever) can be shown with a "show x more" to expand the details.